src/application-status-overlay/ApplicationStatusOverlayProvider.jsx

```diff
@@ -69,6 +69,15 @@ const ApplicationStatusOverlayProvider = ({ children, scrollRefCallback, ...cust
   const registeredVariant = registeredStatusOverlay[lastRegisteredStatusOverlayKey]?.variant;
 
   useEffect(() => {
+    const { head } = document;
+    if (!head.querySelector('link#inert-style')) {
```

Comment:
lets wrap all this link injection under the if on line 82

Reply:
Updated here https://github.com/cerner/terra-application/pull/89/commits/30513765b7611c3c5f527f56573286f521ce9c33.

Reply:
Updated here https://github.com/cerner/terra-application/pull/89/commits/30513765b7611c3c5f527f56573286f521ce9c33.